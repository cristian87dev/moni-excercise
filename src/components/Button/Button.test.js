import React from 'react';
import renderer from 'react-test-renderer';
import { Button, SubmitBtn } from './index';

it('Button renders without crashing', () => {
  const onClick = jest.fn();
  const component = renderer.create(
    <Button type="" onClick={onClick} />,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it('SubmitBtn renders without crashing', () => {
  const component = renderer.create(
    <SubmitBtn />,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
