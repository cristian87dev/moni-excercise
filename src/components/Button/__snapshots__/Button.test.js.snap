// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders without crashing 1`] = `
<button
  className="button basic"
  onClick={[Function]}
  type=""
/>
`;

exports[`SubmitBtn renders without crashing 1`] = `
<button
  className="button submit"
  type="submit"
/>
`;
